charts/ratify/values.yaml

```diff
@@ -15,6 +15,15 @@ notationCerts: []
 cosign:
   enabled: true
   key: ""
+vulnerabilityreport:
+  enabled: true
```

Comment:
i m considering disabling SBOM verifier in the default setup, and user will have to be intentional to enable it and  populate verifier properties.

Reply:
good point. I'll make it false too